packit_service/worker/reporting.py

```diff
@@ -33,6 +33,17 @@ class BaseCommitStatus(Enum):
     error = "error"
 
 
+class DuplicateCheckMode(Enum):
+    """Enum of possible behaviour for handling duplicates when commenting."""
+
+    # Do not check for duplicates
+    do_not_check = auto()
+    # Check only last comment from us for duplicate
+    check_last_comment = auto()
+    # Check the whole comment list for duplicate
+    check_all_comments = auto()
```

Comment:
I am wondering if we want to switch between the modes and don't be consistent but I like it after all..;) really nicely implemented!